rotkehlchen/externalapis/etherscan.py

```diff
@@ -372,6 +374,19 @@ def get_transactions(
                                 transactions=[internal_tx],
                                 relevant_address=None,  # can't know the address here
                             )
+
+                        dbevents = DBHistoryEvents(self.db)  # type: ignore[arg-type]
```

Comment:
instead of having this multiple ignore types for the db why not do a `assert self.db is not None` at the start of the method?

Reply:
Can be also done that way